lang/it.json

```diff
@@ -71,14 +71,14 @@
   "Proportional Serif": "Carattere a spaziatura variabile con grazie (Proportional Serif)",
```

Comment:
Not related to your changes, but noting while I'm here that these are way too long for the captions settings box where they're displayed.

Reply:
Agree, these look to be literal translations :) I'll fix them in another commit using "Serif proporzionale" and "Serif monospazio" labels